Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -614,7 +613,7 @@ private void SaveItemsInTranscation(IDatabaseConnection db, IEnumerable<(BaseIte
             {
                 GetSaveItemCommandText(),
                 "delete from AncestorIds where ItemId=@ItemId"
-            });
+            }).ToList();
```

Comment:
why suddenly `ToList`?

Reply:
Before the function returned a list ( it did `ToList` internally), now it returns an `IEnumerable`

Reply:
But why you need a list here, why `IEnumerable` isn't enough?